test/e2e/nsx_networkinfo_test.go

```diff
@@ -58,12 +58,20 @@ func testCustomizedNetworkInfo(t *testing.T) {
 	// Create customized networkconfig
 	ncPath, _ := filepath.Abs("./manifest/testVPC/customize_networkconfig.yaml")
 	require.NoError(t, applyYAML(ncPath, ""))
-	nsPath, _ := filepath.Abs("./manifest/testVPC/customize_ns.yaml")
-	require.NoError(t, applyYAML(nsPath, ""))
 
-	defer deleteYAML(nsPath, "")
+	ns := fmt.Sprintf("customized-ns-%s", getRandomString())
+	setupTest(t, ns)
+	defer teardownTest(t, ns, defaultTimeout)
 
-	ns := "customized-ns"
+	// Add the required annotation to the namespace
+	namespace, err := testData.clientset.CoreV1().Namespaces().Get(context.TODO(), ns, v1.GetOptions{})
+	require.NoError(t, err)
+	if namespace.Annotations == nil {
+		namespace.Annotations = make(map[string]string)
+	}
+	namespace.Annotations["nsx.vmware.com/vpc_network_config"] = testCustomizedNetworkConfigName
```

Comment:
I think we don't support modify network config annotation after the namespace is created. In this way, we actually create the namespace with default network config instead of testCustomizedNetworkConfigName. You may need to define the network config annotation before creating the namespace.

Reply:
you are right, forget to alter